docs/adopters.md

```diff
@@ -0,0 +1,77 @@
+---
+title: TiKV Adopters
+category: adopters
+---
+
+# TiKV Adopters
+
+This is a list of TiDB adopters in various industries.
```

Comment:
The title said "TiKV Adopters" but the description becomes "TiDB adopters". May cause confusion.